chrome/views/popupView.js

```diff
@@ -191,6 +254,9 @@ GmailToTrello.PopupView.prototype.onResize = function() {
 };
 
 GmailToTrello.PopupView.prototype.resetDragResize = function() {
+    if (this.$popup.hasClass('ui-resizable')) {
+      this.$popup.draggable('destroy').resizeable('destroy');
+    }
```

Comment:
This is a bug fix as well. The system was creating multiple resizables on top of one another. So now it destroys the existing resizable every time it adds another one on.